torch/fx/_symbolic_trace.py

```diff
@@ -532,7 +532,8 @@ def trace(self, root: Union[torch.nn.Module, Callable], concrete_args: Optional[
         else:
             self.root = torch.nn.Module()
             fn = root
-        self.graph = Graph()
+
+        self.graph = Graph(tracer_cls=self.__class__)
```

Comment:
You may want to wrap self.__class__ with Optional to silence mypy warning: https://github.com/pytorch/pytorch/runs/3223778148